docker-compose.yaml

```diff
@@ -19,7 +19,7 @@ services:
         volumes:
             - pg-data:/var/lib/postgresql/data:rw
         ports:
-            - "5432:55432"
+            - "55432:5432"
```

Comment:
@sstok it's HOST:CONTAINER, not CONTAINER:HOST :laughing:

Reply:
Man I feel stupid now 🤦‍♂️ 

![Why, why must life be so hard](https://media.giphy.com/media/GOkF76siWniJG/giphy.gif)

Reply:
FWIW, CONTAINER:HOST makes sense to me too, HOST:CONTAINER feels backwards.